fe/fe-core/src/test/java/com/starrocks/planner/QueryPlannerTest.java

```diff
@@ -165,6 +165,16 @@ public void testSqlBlackList() throws Exception {
         StatementBase statement = SqlParser.parse(sql, connectContext.getSessionVariable().getSqlMode()).get(0);
         StmtExecutor stmtExecutor2 = new StmtExecutor(connectContext, statement);
         stmtExecutor2.execute();
+        Assert.assertEquals("Access denied; This sql is in blacklist, please contact your admin",
+                connectContext.getState().getErrorMessage());
+        connectContext.getState().setError("");
+
+        String sqlWithLineSeparators = "select k1 \n" +
+                " from \n" +
+                " test.baseall";
+        StatementBase statement1 = SqlParser.parse(sqlWithLineSeparators, connectContext.getSessionVariable().getSqlMode()).get(0);
+        StmtExecutor stmtExecutor4 = new StmtExecutor(connectContext, statement1);
+        stmtExecutor4.execute();
```

Comment:
Could you add some sql-tester tests?

Reply:
testSqlBlackList  contains sqlWithNoLineSeparators and sqlWithLineSeparators Test Cases